maturity-level-one/src/Controllers/PlayersController.cs

```diff
@@ -81,7 +77,7 @@ public async Task<IActionResult> Create(Player player)
         }
 
         [HttpPut("{id}")]
-        [SwaggerOperation("update-player")]
+        [SwaggerOperation("Players_Update")]
```

Comment:
Should be UpdateFull to align with operation name?

Reply:
Yes, oversight on my part. Will change